bin/_git-fetch

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# -*- mode: sh; indent-tabs-mode: nil; sh-basic-offset: 4 -*-
+# vim: autoindent tabstop=4 shiftwidth=4 expandtab softtabstop=4 filetype=bash
+
+# fetch git updates for all remotes
+
+. /etc/sysconfig/crucible
+
+if [ -z "${CRUCIBLE_HOME}" ]; then
+    echo "CRUCIBLE_HOME not defined, exiting."
+    exit 1
+fi
+
+if [ ! -e ${CRUCIBLE_HOME} ]; then
+    echo "Could not find ${CRUCIBLE_HOME}, exiting."
+    exit 1
+fi
+
+source ${CRUCIBLE_HOME}/bin/base
+
+LOCK_FILE="${UPDATE_STATUS_FILE}.lock"
+
+if [ -e "${LOCK_FILE}" ]; then
```

Comment:
just quit ?

Reply:
Yeah, this is just to prevent the updater from running multiple times simultaneously.  Since this is being spun off in the background it is possible for multiple commands to be run while the original backgrounded process is still running so this prevents additional copies from executing.  Having multiple copies executing isn't really a very bad thing, it's just very inefficient and sending unecessary traffic to the upstream repo.